added file named aldhanekaa-get-number-of-words-and-number-of-sentences.js

Aldhanekaa opened the pull request:
added file `aldhanekaa get number of words and number of sentences.js`
If you are not satisfied with this file or algorithm,  I can create a new one!

Aldhanekaa committed:
added file aldhanekaa get number of words and number of sentences.js

Aldhanekaa committed:
change aldhanekaa get number of words and number of sentences.js become aldhanekaa-get-number-of-words-and-number-of-sentences.js

JazibJafri commented:
@Aldhanekaa function name should match with the function name in filename and also add a single line description of the function

Aldhanekaa committed:
the function name changed to a name like the file name, and add a single line description of the function

jazibjafri commented on src/javascript/aldhanekaa-get-number-of-words-and-number-of-sentences.js:
```diff
@@ -0,0 +1,18 @@
+let story = 'Last weekend, I took literally the most beautiful bike ride of my life. The route is called "The 9W to Nyack" and it actually stretches all the way from Riverside Park in Manhattan to South Nyack, New Jersey. It\'s really an adventure from beginning to end! It is a 48 mile loop and it basically took me an entire day. I stopped at Riverbank State Park to take some extremely artsy phot ...
+
+// function to get number of words and number of sentences from its parameter, then return an object 
+function getNumberOfWordsAndNumberOfSentences(str) {
```
Function is not the same as in the filename, rename the file to `<your-user-name>-getNumberOfWordsAndNumberOfSentences.js`

Aldhanekaa committed:
Rename aldhanekaa-get-number-of-words-and-number-of-sentences.js 

Rename aldhanekaa-get-number-of-words-and-number-of-sentences.js to Aldhanekaa-getNumberOfWordsAndNumberOfSentences.js
